Code/GraphMol/Substruct/SubstructMatch.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include "SubstructMatch.h"
 #include "SubstructUtils.h"
+#include <boost/serialization/array_wrapper.hpp>
```

Comment:
Since I think this is really only relevant for boost v1.64 you should almost certainly check the boost version here with something like (not tested):
```
#if BOOST_VERSION == 106400
#include <boost/serialization/array_wrapper.hpp>
#endif
```